neurokit2/ecg/ecg_phase.py

```diff
@@ -30,12 +30,12 @@ def ecg_phase(ecg_cleaned, rpeaks=None, delineate_info=None, method='peak',  sam
 
         - *"ECG_Phase_Artrial"*: cardiac phase, marked by "1" for systole
           and "0" for diastole.
-        - *"ECG_Ventricular_Phase"*: cardiac phase, marked by "1" for systole
+        - *"ECG_Phase_Ventricular"*: cardiac phase, marked by "1" for systole
           and "0" for diastole.
-          *"ECG_Atrial_PhaseCompletion"*: cardiac phase (atrial) completion,
+          *"ECG_PhaseCompletion_Atrial"*: cardiac phase (atrial) completion,
           expressed in percentage (from 0 to 1), representing the stage of the
           current cardiac phase.
-          *"ECG_Ventricular_PhaseCompletion"*: cardiac phase (ventricular)
+          *"ECG_PhaseCompletion_Ventricular"*: cardiac phase (ventricular)
```

Comment:
maybe also ECG_PhaseCompletion_Ventricular to ECG_Phase_Ventricular_Completion. It's quite wordy but makes more logical sense I think?

Reply:
I think I named it _PhaseCompletion to standardize it with RSP_PhaseCompletion and true that to make it less wordy. But yeah why not. If that makes it clearer 👍